.github/workflows/ci.yml

```
@@ -23,6 +23,12 @@ jobs:
     - name: Run tests
       run: python -m manage test
 
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v3
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        files: ./coverage.xml
```

Let's keep the inputs ordered:
```suggestion
        files: coverage.xml
        token: ${{ secrets.CODECOV_TOKEN }}
```